examples/federation-workshop/cluster-2/bundle/Endpoint.yaml

```diff
@@ -0,0 +1,23 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+apiVersion: v1
```

Comment:
We’ll need to expose SPIRE server bundle endpoints for the clusters to federate.

Federation is one of the many topologies that we can have with SPIRE.